spyder/plugins/editor/widgets/editor.py

```diff
@@ -1568,6 +1571,22 @@ def close_all_but_this(self):
         for i in range(0, self.get_stack_count()-1  ):
             self.close_file(0)
 
+    def sort_file_tabs_alphabetically(self):
+        """Sort open tabs alphabetically."""
+        while self.sorted() is False:
+            for i in range(0, self.tabs.tabBar().count()):
+                if(self.tabs.tabBar().tabText(i) >
+                   self.tabs.tabBar().tabText(i+1)):

```

Comment:
This needs to be

```python
if (self.tabs.tabBar().tabText(i) >
        self.tabs.tabBar().tabText(i + 1)):
```

The second line is indented 8 spaces with respect to the `if`.

Reply:
More specifically, what @ccordoba12 described is our project's style convention for this situation.